array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,21 @@
+/* exported tail */
+
+// We need a storage for the new array.
+// We need to check if the data is first or not.
+// If not, we store the data into the new array.
+// Return the new array.
+
+function tail(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i >= 1) {
```

Comment:
Instead of including a conditional in every iteration of the loop to check if the current index is not 0, you should start your loop at 1 so you don't need to worry about it.